### DISCUSSION TITLE
Potentially including 'Live' photo images shown in html export

### DISCUSSION THREAD

**Discussion by BeefyResistor** [ReagentX/imessage-exporter#111] [Ideas]
Love the project overall, no complaints. Thanks for putting the whole thing together.

I noticed that 'Live' images is one format that doesn't get picked up in the html export.

I'm guessing it's a ball-ache to do considering the files are saved as video files (or something like that, if I recall), and considering that most people who send Live photos seem to do it by accident anyway!

Just wondered what the thinking is around this.

Perhaps if the file is one that will result in a format change, said file could be included in the same directory as its converted version? That way it allows for a slightly more comprehensive backup.

Thanks again.

**Comment by ReagentX** [comment]
The reason this doesn't happen currently is that those HEIC/HEIF files cannot be displayed in Safari, so I flatten them to a standard JPEG. 

You make a good point about Live Photos though, perhaps those should get converted to a different type of portable format (or not converted at all). I like the idea of copying both the "compatible" version and the "original" one; that might be the best way to move forward here.

**Comment by ReagentX** [comment]
Support for different attachment management methods was added in #134 for #125.
